staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/authentication.go

```diff
@@ -22,28 +22,19 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/apiserver/pkg/admission/options"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 )
 
-type AuthenticationInfoResolverWrapper func(AuthenticationInfoResolver) AuthenticationInfoResolver
-
-type AuthenticationInfoResolver interface {
-	ClientConfigFor(server string) (*rest.Config, error)
-}
-
-type AuthenticationInfoResolverFunc func(server string) (*rest.Config, error)
-
-func (a AuthenticationInfoResolverFunc) ClientConfigFor(server string) (*rest.Config, error) {
-	return a(server)
-}
-
 type defaultAuthenticationInfoResolver struct {
 	kubeconfig clientcmdapi.Config
 }
 
-func NewDefaultAuthenticationInfoResolver(kubeconfigFile string) (AuthenticationInfoResolver, error) {
+// NewDefaultAuthenticationInfoResolver genearates an AuthenticationInfoResolver
+// that builds rest.Config based on the kubeconfig file.
```

Comment:
Clarify that the kubeconfigFile variable is a loaded kube config file, not a file path.

Reply:
It actually is the path.